.gitignore

```diff
@@ -1,20 +1,29 @@
-*#
+# Python
 *.py[co]
+.virtualenv
+_venv
+.pytest_cache/
+
+# Node
+node_modules
+
+# WPT repo stuff
+/MANIFEST.json
+
+testharness_runner.html
+!/tools/wptrunner/wptrunner/testharness_runner.html
+
+_certs
+config.json
+
+# Various OS/editor specific files
+*#
 *.sw[po]
 *~
-MANIFEST.json
-!content-security-policy/support/manifest.json
-!payment-handler/manifest.json
 \#*
-_certs
-.virtualenv
-config.json
-node_modules
 scratch
-testharness_runner.html
 webdriver/.idea
 .vscode/
 .DS_Store
 *.rej
-_venv
 webdriver/.cache
```

Comment:
What even is this? Should this be in the previous section?

Reply:
pytest junk

Reply:
oh is this where `.pytest_cache` comes from? that, but renamed?

Reply:
Yeah it seems that the name changed at some point, so maybe this rule should be updated.